lib/ASRouter.jsm

```diff
@@ -1767,7 +1767,62 @@ class _ASRouter {
           csp: null,
         });
         break;
+      case ra.ENABLE_FIREFOX_MONITOR:
+        const {
+          utm_term,
+          utm_source,
+          utm_campaign,
+          entrypoint,
+          form_type,
+        } = action.data;
+        const { flowBeginTime, flowId } = await this.fetchFlowParams({
+          entrypoint,
+          utm_term,
+          form_type,
+        });
+        const monitorURL = `https://monitor.firefox.com/oauth/init?utm_source=${utm_source}&utm_campaign=${utm_campaign}&form_type=${form_type}&entrypoint=${entrypoint}&flowBeginTime=${flowBeginTime}&flowId=${flowId}`;
+        // We want to replace the current tab.
+        target.browser.ownerGlobal.openLinkIn(monitorURL, "current", {
+          private: false,
+          triggeringPrincipal: Services.scriptSecurityManager.createNullPrincipal(
+            {}
+          ),
+          csp: null,
+        });
+        break;
+    }
+  }
+
+  async fetchFlowParams({ utm_term, entrypoint, form_type }) {
+    let result = {};
+    const fxaEndpoint = Services.prefs.getStringPref(
```

Comment:
Do we want to use this Firefox accounts endpoint or rely on the snippets service to provide one?

Reply:
Is this targeting Firefox 70? Over in https://bugzilla.mozilla.org/show_bug.cgi?id=1568949 and https://bugzilla.mozilla.org/show_bug.cgi?id=1556276, there is talk of having newtab automatically respect the `identity.fxaccounts.remote.root` values, it seems like this should use that same code, which I imagine is bbased off of `identity.fxaccounts.remote.root`?

Reply:
This will be uplifted to 69.